inc/views/nav_walker.php

```diff
@@ -12,6 +12,7 @@
 
 use HFG\Core\Components\Nav;
 use Neve\Core\Dynamic_Css;
+use function _PHPStan_bcbc46924\RingCentral\Psr7\str;
```

Comment:
Was it added by mistake?

Reply:
yes, it was added by the IDE, thank you for catching this. I've fixed the extra import.